src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentOneDay = 40;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
```

Comment:
Consider using more descriptive variable names, such as 'sevenDaysDiscount' instead of 'discountSevenDays'.